File: src/Generated20260405.go
package main

import "fmt"

type Generated20260405 struct {
    // Generated struct 20260405_155531_546507
    Value int
    Created string
}

func (c *Generated20260405) Process() int {
    result := c.Value
    for i := 0; i < 9; i++ {
        result += i * 4
    }
    return result
}

func main() {
    obj := Generated20260405{Value: 957, Created: "20260405_155531_546507"}
    fmt.Println(obj.Process())
}

File: src/Generated20260405.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260405 {
public:
    // Generated class 20260405_155310_140688
    int value = 272;
    string created = "20260405_155310_140688";
    
    int process() {
        int result = value;
        for (int i = 0; i < 17; i++) {
            result += i * 9;
        }
        return result;
    }
};

int main() {
    Generated20260405 obj;
    cout << obj.process() << endl;
    return 0;
}